CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2.4.2] - UNRELEASED
```

Comment:
I would say this is more of added functionality than a bug fix (do you agree?) so I think it should be version 2.5.0 based on: https://semver.org/

Reply:
I agree they are not bug fixes. The question I have now about my message below is if we consider these points as backwards-compatible. Functionally the API is the same and produces dashboards that look the same. I just had to modify a test or two.

Reply:
Is there no way to add this functionality as an option rather than the default so that it remains backwards compatible?

Reply:
I don't think it's worth it. There was a technical issue with, before, using pixels as the metric for slicing up the boxes (since it was _always_ in half) but the new param uses percentage to insert the boxes next to a pre-existing box

Reply:
ok got it 👍

Reply:
@Kully did you get a chance to update this to `2.5.0` and update `plotly/version.py` as well?

Reply:
no thank you for reminding me